setup.py

```diff
@@ -10,7 +10,10 @@
     url='https://github.com/davidmuller/aws-requests-auth',
     description='AWS signature version 4 signing process for the python requests module',
     long_description='See https://github.com/davidmuller/aws-requests-auth for installation and usage instructions.',
-    install_requires=['requests>=0.14.0'],
+    install_requires=[
+        'botocore>=1.7.8',
```

Comment:
It's a bit odd, but we don't require `botocore` as a strict dependency of the package:

> [botocore (the core functionality of boto3) is not a strict requirement of aws-requests-auth, but we do provide some convenience methods if you'd like to use botocore to automatically retrieve your AWS credentials for you.](https://github.com/DavidMuller/aws-requests-auth#using-boto-to-automatically-gather-aws-credentials)